<?xml version="1.0"?>
<launch>
  <node if="false" pkg="graph_core" type="highdof_test" name="$(anon mixed_h)" output="screen"  launch-prefix="gnome-terminal --geometry 80x43+55+14 --">
    <rosparam  command="load" file="$(find graph_core)/config/high_dof.yaml" />
    <param  name="tube_sampler" value="true" />
    <param  name="forgetting_factor" value="0.999" />
    <param  name="tube_radius" value="0.02" />
  </node>
  <node if="true" pkg="graph_core" type="highdof_test" name="$(anon informed_h)" output="screen"  launch-prefix="gnome-terminal --geometry 80x43+55+14 --">
    <rosparam  command="load" file="$(find graph_core)/config/high_dof.yaml" />
    <param  name="tube_sampler" value="false" />
    <param  name="forgetting_factor" value="0.999" />
    <param  name="tube_radius" value="0.02" />
  </node>
  <node pkg="graph_core" type="highdof_test" name="$(anon warp_h)" output="screen"  launch-prefix="gnome-terminal --geometry 80x43+55+14 --">
    <rosparam  command="load" file="$(find graph_core)/config/high_dof.yaml" />
    <param  name="tube_sampler" value="false" />
    <param  name="warp" value="true" />
    <param  name="forgetting_factor" value="0.999" />
    <param  name="tube_radius" value="0.02" />
  </node>
</launch>
